src/controllers/requestController.js

```diff
@@ -31,4 +31,24 @@ export default class RequestController {
       errorResponse(res, { code: 500, message: e.message });
     }
   }
+
+  /**
+   *  creates a one way trip request
+   * @static
+   * @param {Request} req - The request from the endpoint.
+   * @param {Response} res - The response returned by the method.
+   * @returns { JSON } - A JSON object containing success or failure details.
+   * @memberof RequestController
+   */
+  static async oneWayTripRequest(req, res) {
+    try {
+      const { body } = req;
+      const { requester } = req;
+      const oneWayTrip = await createTripRequest({ ...body });
+      delete oneWayTrip.returnDate;
```

Comment:
Shouldn't the delete happen before the creation on line 47?

Reply:
resolved @amarachukwu-agbo